### PR TITLE
[v2.8] Removed support for CIS scan profile and benchmarks for versions prior to k8s 1.25

### PR DESCRIPTION
**Pull request by chiukapoor**
## Issue:
* Fixes https://github.com/rancher/cis-operator/issues/289

## Solution
* Removed support for CIS scan profile and benchmarks for versions prior to k8s 1.25
```
1.20
1.23
1.24
1.5
1.6
```
* Bump `rancher-cis-benchmark` to `v5.2.0-rc1`

### CODE REVIEW

**Review by adamkpickering — APPROVED** [review]
LGTM once the merge conflicts are resolved.

File: charts/rancher-cis-benchmark/5.2.0-rc1/app-readme.md
Comment (andypitcher) [inline charts/rancher-cis-benchmark/5.2.0-rc1/app-readme.md]:
```suggestion
For more information on how to use the feature, refer to our [docs](https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/cis-scan-guides).
```
Update link to `latest` documentation.

File: charts/rancher-cis-benchmark/5.2.0-rc1/app-readme.md
Comment (andypitcher) [inline charts/rancher-cis-benchmark/5.2.0-rc1/app-readme.md:6]:
```suggestion

## CIS Kubernetes Benchmark support

| Source | Kubernetes distribution                                                                                                                                                                              | scan profile        | Kubernetes versions |
|--------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|---------------------|
| CIS    | any  | [cis-1.7](https://github.com/rancher/security-scan/tree/master/package/cfg/cis-1.7) | v1.25 |
| CIS    | any | [cis-1.8](https://github.com/rancher/security-scan/tree/master/package/cfg/cis-1.8) | v1.26+ |
| CIS    | rke  | [rke-cis-1.7-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.7-permissive) | rke1-v1.25     |
| CIS    | rke  | [rke-cis-1.7-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.7-permissive) | rke1-v1.25     |
| CIS    | rke  | [rke-cis-1.7-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.7-hardened) | rke1-v1.25     |
| CIS    | rke  | [rke-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.8-permissive) | rke1-v1.26+     |
| CIS    | rke  | [rke-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.8-hardened) | rke1-v1.26+     |
| CIS    | rke2  | [rke2-cis-1.7-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/rke2-cis-1.7-permissive) | rke2-v1.25     |
| CIS    | rke2  | [rke2-cis-1.7-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/rke2-cis-1.7-hardened) | rke2-v1.25     |
| CIS    | rke2  | [rke2-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/rke2-cis-1.8-permissive) | rke2-v1.26+     |
| CIS    | rke2  | [rke2-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/rke2-cis-1.8-hardened) | rke2-v1.26+     |
| CIS    | k3s  | [k3s-cis-1.7-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/k3s-cis-1.7-permissive) | k3s-v1.25     |
| CIS    | k3s  | [k3s-cis-1.7-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/k3s-cis-1.7-hardened) | k3s-v1.25     |
| CIS    | k3s  | [k3s-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/master/package/cfg/k3s-cis-1.8-permissive) | k3s-v1.26+     |
| CIS    | k3s  | [k3s-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/k3s-cis-1.8-hardened) | k3s-v1.26+     |
| CIS    | eks  | eks-1.2.0 | eks    |
| CIS    | aks  | aks-1.0 | aks     |
| CIS    | gke  | gke-1.2.0 | gke     |
```
Adding more details in the chart info about supported versions, as it's done on kube-bench: https://github.com/aquasecurity/kube-bench/blob/main/docs/platforms.md .

**Review by nicholasSUSE — APPROVED** [review]
Hello, we have improved the Charts documentation regarding Pull Request Rules.

Please take a look at it before making any new Pull Requests.
We are still approving Pull Requests and starting to inform all teams, soon we will require the standard in the documentation.

https://github.com/rancher/charts?tab=readme-ov-file#pull-request-rules

**Review by andypitcher — APPROVED** [review]

**Review by vardhaman22 — APPROVED** [review]

**Pull request merged**
